casper/src/main/scala/coop/rchain/casper/BlockCreator.scala

```diff
@@ -66,14 +66,18 @@ object BlockCreator {
       import cats.instances.list._
       for {
         tipHashes             <- Estimator[F].tips(dag, genesis)
+        _ <- Log[F].info(s"Creating block with tip hashes ${tipHashes.map(PrettyPrinter.buildString)}")
```

Comment:
Maybe use one log line "Creating block with...." with all the content instead of 4 lines?

Reply:
TBH, personally I prefer splitting them which we can also see the execution time for each step.